shop/urls.py

```diff
@@ -4,7 +4,9 @@
 
 urlpatterns = [
     path('', views.ShopView.as_view(), name='product_list'),
+    path('filter', views.ShopView.as_view(), name='product_list'),
     path('category/<int:pk>/', views.ShopView.as_view(), name='category'),
+    path('category/<int:pk>/filter', views.ShopView.as_view(), name='category'),
```

Comment:
giống nhau như này thì gộp lại 1 path thôi e, bên trong views sẽ xử lý luôn cả phần search nhé

Reply:
em muốn tách nhiều url, mình có thể copy link gửi cho người khác, thì họ sẽ ra được luôn cái người gửi đang xem
ví dụ : khi truy /category/10/filter?sorting=price_low_to_high thì mình có luôn danh sách với id_category = 10 và giá tăng dần

Reply:
em sửa rồi ạ
về chức năng search, em tham khảo mấy trang bán hàng, họ toàn sử lý 1 url riêng(http:port/sreach?keywork_search, em đang làm như này)